### PR TITLE
Changes to +context, boosters and custom language flags

### PR DESCRIPTION
**Pull request by ImRodry**
The changes are explained in the description of each commit.

### CODE REVIEW

File: commands/context.js [hunk 1]
```diff
@@ -508,7 +507,7 @@ async function showInfo(executedBy, strings, message, args) {
                     { name: "Edit", value: strings.info.edit },
                     { name: "View", value: strings.info.view },
```
Comment (QkeleQ10) [inline commands/context.js:508]: Please replace `view` with `link` here, since you've changed the string name.

**Review by QkeleQ10 — CHANGES_REQUESTED** [review]

File: commands/context.js (hunk 1 above)
Comment (ImRodry) [reply commands/context.js:508]: oh yikes, lemme fix

File: config.json [hunk 2]
```diff
@@ -7,18 +7,18 @@
...
     {"name": "Bulgarian", "emoji": "🇧🇬", "colour": "#e02222", "code" : "bg", "flag": "https://crowdin.com/images/flags/%code%.png"},
     {"name": "Chinese Simplified", "emoji": "🇨🇳", "colour": "#21de70", "code": "zhcn", "flag": "https://crowdin.com/images/flags/%code%.png"},
     {"name": "Chinese Traditional", "emoji": "🇭🇰", "colour": "#21de70", "code": "zhtw", "flag": "https://crowdin.com/images/flags/%code%.png"},
     {"name": "Croatian", "emoji": "🇭🇷", "code" : "hr", "flag": "https://crowdin.com/images/flags/%code%.png"},
     {"name": "Czech", "emoji": "🇨🇿", "colour": "#e02222", "code": "cs", "flag": "https://crowdin.com/images/flags/%code%.png"},
     {"name": "Danish", "emoji": "🇩🇰", "colour": "#e02222", "code": "da", "flag": "https://crowdin.com/images/flags/%code%.png"},
     {"name": "Dutch", "emoji": "🇳🇱", "colour": "#21de70", "code": "nl", "flag": "https://crowdin.com/images/flags/%code%.png"},
-    {"name": "Emoji", "emoji": "😂", "code" : "em", "flag": "https://i.imgur.com/oRoCMON.png"},
+    {"name": "Emoji", "emoji": "😂", "code" : "em", "flag": "https://github.com/QkeleQ10/hypixel-translators-bot-discord/blob/master/flags/%code%.png?raw=true"},
```
Comment (QkeleQ10) [inline config.json:21]: Or just make it "./flags/%code%" 😄

File: config.json [hunk 3]
```diff
@@ -29,7 +29,7 @@
     {"name": "Hungarian", "emoji": "🇭🇺", "code" : "hu", "flag": "https://crowdin.com/images/flags/%code%.png"},
     {"name": "Indonesian", "emoji": "🇮🇩", "code" : "id", "flag": "https://crowdin.com/images/flags/%code%.png"},
     {"name": "Irish", "emoji": "🇮🇪", "code" : "gaie", "flag": "https://crowdin.com/images/flags/%code%.png"},
-    {"name": "Inhale Banana", "emoji": "🍌", "code" : "ib", "flag": "https://i.imgur.com/Moavpb7.png"},
+    {"name": "Inhale Banana", "emoji": "🍌", "code" : "ib", "flag": "https://github.com/QkeleQ10/hypixel-translators-bot-discord/blob/master/flags/%code%.png?raw=true"},
```
Comment (QkeleQ10) [inline config.json:32]: "./flags/%code%"

**Review by QkeleQ10 — APPROVED** [review]

**Pull request merged**